torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2993,17 +3059,55 @@ void FoldQuantizedPrepackingOps(Module& module) {
   PrePackingOpsFolder(module, filter_fn, "quantized");
 }
 
+void RemoveRedundantQuantizeOps(
```

Comment:
you can split this to a separate PR

Reply:
I feel this is still related to the change of replicating choose_qparams and quant. We shouldn't have a case of extra choose_qparams and quant nodes in the graph because it will affect perf. The test checks for this as well.
If you think splitting it helps in some other way then let me know.

Reply:
it's a different pass though, you can still land them together even after you split. 
But I don't have a strong preference here.